main.nf

```diff
@@ -283,6 +284,69 @@ process intersect_files {
     """
 }
 
+
+vep_infiles = masked_beds_vep.mix(vcfs)
+
+// TODO: Figure out running characteristics
+process variant_effect_predictor {
+    input:
+        file infile from vep_infiles
+    output:
+        file '*.vep'
+
+    publishDir params.outdir, mode: 'copy'
+
+    // We only need one core for this part
+    if ( nextflow_running_as_slurmjob() ) {
+        executor 'local'
+    }
+    else {
+        executor 'slurm'
+        queue 'core'
+        time params.short_job
```

Comment:
`time "$params.short_job"` ? Not sure what's meant. Is this a Groovy variable called `params.short_job` or is it a shell variable called `params` (with value `$params`)?  A `$` seems to be missing. I'm not Groovy enough...